src/Table/Table.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { StyledTable } from "./table.styles";
+
+const tableWithDefaults = props => {
```

Comment:
I would go an explicitly define the arguments that are coming as props.
This way we can have Type checking using React `prop-types`.

Reply:
We have typechecking from prop-types (see below comment)

Reply:
Now i see.